Author: Daniel Baumann <daniel@debian.org>
Description: Removing pdf documents without source.

diff -Naurp vmware-view-open-client.orig/Makefile.in vmware-view-open-client/Makefile.in
--- vmware-view-open-client.orig/Makefile.in	2009-04-16 04:03:24.000000000 +0000
+++ vmware-view-open-client/Makefile.in	2009-04-17 18:34:21.000000000 +0000
@@ -1290,7 +1290,7 @@ dist_noinst_HEADERS := lib/open-vm-tools
 	lib/bora/include/vmlocale.h lib/bora/include/vthread.h \
 	lib/bora/include/vthreadBase.h lib/libp11/include/pkcs11.h \
 	extraTranslations.hh
-dist_pdf_DATA := doc/View_Client_Admin_Guide.pdf
+#dist_pdf_DATA := doc/View_Client_Admin_Guide.pdf
 noinst_LIBRARIES := libDict.a libErr.a libFile.a libMisc.a libPanic.a \
 	libPanicDefault.a libString.a libStubs.a libUnicode.a \
 	libUser.a libAsyncSocket.a libBasicHttp.a libLog.a \
