Author: Daniel Baumann <daniel@debian.org>
Description: Removing pdf documents without source.

diff -Naurp vmware-view-open-client.orig/Makefile.in vmware-view-open-client/Makefile.in
--- vmware-view-open-client.orig/Makefile.in	2009-10-02 20:38:31.000000000 +0000
+++ vmware-view-open-client/Makefile.in	2009-10-02 20:55:41.000000000 +0000
@@ -1599,7 +1599,7 @@ dist_noinst_HEADERS := lib/open-vm-tools
 	lib/bora/include/vthreadBase.h lib/libp11/include/pkcs11.h \
 	extraTranslations.hh
 dist_noinst_SCRIPTS := cocoa/pkg-dmg
-dist_pdf_DATA := doc/View_Client_Admin_Guide-en.pdf
+#dist_pdf_DATA := doc/View_Client_Admin_Guide-en.pdf
 noinst_LIBRARIES := libDict.a libErr.a libFile.a libMisc.a libPanic.a \
 	libPanicDefault.a libString.a libStubs.a libUnicode.a \
 	libUser.a libAsyncSocket.a libBasicHttp.a $(am__append_1) \
diff -Naurp vmware-view-open-client.orig/doc/Makefile.inc vmware-view-open-client/doc/Makefile.inc
--- vmware-view-open-client.orig/doc/Makefile.inc	2009-10-02 20:38:31.000000000 +0000
+++ vmware-view-open-client/doc/Makefile.inc	2009-10-02 20:54:36.000000000 +0000
@@ -18,7 +18,7 @@
 # 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA.
 ################################################################################
 
-dist_pdf_DATA += doc/View_Client_Admin_Guide-en.pdf
+#dist_pdf_DATA += doc/View_Client_Admin_Guide-en.pdf
 #dist_pdf_DATA += doc/View_Client_Admin_Guide-de.pdf
 #dist_pdf_DATA += doc/View_Client_Admin_Guide-fr.pdf
 #dist_pdf_DATA += doc/View_Client_Admin_Guide-ja.pdf
